CHANGELOG

```diff
@@ -2,6 +2,8 @@
 = ActiveMerchant CHANGELOG
 
 == HEAD
+
+== Version 1.136.0 (May 1, 2024)
```

Comment:
Did you release a new version?

Reply:
It is an AC on the ticket to do so. I haven't pushed the tag or done the release yet though. I was going to do that in my CAB release window.

Reply:
Ah gotcha. You should probably cut a new version before merging this and then have this in it's own

Reply:
Yeah that makes sense to me I'll update before merging this change.